streams/core.py

```diff
@@ -59,6 +59,13 @@ def __init__(self, child=None, children=None, **kwargs):
             if child:
                 child.parents.append(self)
 
+    def __str__(self):
+        s = self.__class__.__name__
+        for m in ['func', 'predicate']:
+            if hasattr(self, m):
+                s += ', {}'.format(getattr(self, m).__name__)
+        return s
```

Comment:
We might also consider adding `func=` or `predicate=` and then extending this list to other common-and-easy-to-render attributes like `n` and `interval`